gaia-header.js

```diff
@@ -761,9 +755,11 @@ module.exports = component.register('gaia-header', {
   }
 
   [action=close] .action-button:before { content: 'close' }
-  [action=back] .action-button:before { content: 'back' }
   [action=menu] .action-button:before { content: 'menu' }
 
+  [action=back]:-moz-dir(ltr) .action-button:before { content: 'back' }
+  [action=back]:-moz-dir(rtl) .action-button:before { content: 'forward' }
```

Comment:
It would be awesome if you could make that change in github.com/gaia-components/gaia-component. So we get the benefits everywhere. It would be good to fallback to the mutation observe if the localised event isn't going to fire. Not sure the best way to make that decision though. We could check for presence of `navigator.mozL10n`; but then that could be lazy-loaded so perhaps not that reliable. I'm easy :)